pkg/proto/go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/golang/protobuf v1.5.4
 	github.com/google/gofuzz v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/planetscale/vtprotobuf v0.6.1-0.20240319094008-0393e58bdf10
```

Comment:
where is this new dependency needed?

Reply:
It's declared here: https://github.com/DataDog/datadog-agent/blob/ba08dfcc0bb4b569fc5a59e65652b46eeeba42a4/internal/tools/proto/tools.go#L23